### PR TITLE
api: training request

### LINKED ISSUE #130: feat: integrate stores to modelTrain flow
### Task description

- [x] create modelStore, trainingRequest Store @techyNonso 
- [x] put the dummy data in `init` , so that we can verify the UI works alright @techyNonso 
- [x] create apis to add trainingRequest, models to DB (refer prisma/seed.ts). Refer src/app/api/prisma/**.ts @Sheriff-Oladimeji 
- [x] create the apis to createTrainingRequest, models. @Sheriff-Oladimeji 
- [ ] integrate the apis:  similar to userStore with init, sync @Sheriff-Oladimeji 

### Related Issues

_No response_

### PR DESCRIPTION
**Pull request by Sheriff-Oladimeji**
### Description

- [x] Resolves #130 

### Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [x] I have added suitable unit tests (if applicable).

To be reviewed by: @Nasfame @techyNonso

### CODE REVIEW

File: src/app/api/prisma/add_training_request/route.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    })
+    return NextResponse.json(new_training_request, { status: 200 })
+  } catch (error) {
+return NextResponse.json({ message: 'Error creating Training Request', error, status: 500 })
+  }
+}
+
+export async function GET() {
+  try {
+    const models = await prisma.model.findMany()
```
Comment (Nasfame) [inline src/app/api/prisma/add_training_request/route.ts:18]: this should be Training Request.many

Comment (Nasfame) [reply src/app/api/prisma/add_training_request/route.ts:18]: Have u tested these apis with postman or a rest client?

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: src/app/api/prisma/add_training_request/route.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ return NextResponse.json({ message: 'Error creating Training Request', error, st
 
 export async function GET() {
   try {
-    const models = await prisma.model.findMany()
+    const models = await prisma.trainingRequest.findMany()
```
Comment (Nasfame) [inline src/app/api/prisma/add_training_request/route.ts:18]: its meant to be const trainingRequests =

**Review by Nasfame — CHANGES_REQUESTED** [review]

**Comment by Nasfame** [comment]
@Sheriff-Oladimeji kindly update ur branch

File: src/app/api/prisma/add_training_request/route.ts
Comment (Nasfame) [inline src/app/api/prisma/add_training_request/route.ts:1]: Not exactly what's required, but its alright.

**Review by Nasfame — APPROVED** [review]
LGTM

File: src/app/dashboard/settings/profile/components/EditProfile.tsx [hunk 3]
```diff
@@ -98,17 +98,10 @@ const EditProfile = ({ onclick }) => {
...
+          <button className=" bg-transparent border border-primary_8 rounded-full px-4 py-3 text-[#F5F5F5] w-[140px] text-center">
             Cancel
           </button>
-          <button
-            className=" bg-primary_10 rounded-full px-4 py-2 w-[140px] text-center"
-            onClick={onclick}
-            // TODO: fixme this is not activated. Use userStore.syncDB to save to DB. And pass false as second argument to userStore.setUser
-          >
+          <button className=" bg-primary_10 rounded-full px-4 py-3 w-[140px] text-center">
             Save changes
```
Comment (Nasfame) [inline src/app/dashboard/settings/profile/components/EditProfile.tsx:105]: Explain the change @Sheriff-Oladimeji . Shouldn't u be able to reset the changes?

**Review by Nasfame — COMMENTED** [review]
@techyNonso kindly review this code

**Review by techyNonso — APPROVED** [review]
Am approving this however , hard coded colors is wrong

**Pull request merged**